lib/angular/index.js

```diff
@@ -0,0 +1,11 @@
+var modrain = require('../modrain');
+
+
+//TODO: This should probably be a separate module.
```

Comment:
IMHO We trying here to build some kind of micro framework. 
Best to introduce services exposing main object for each UI technology and allow to enable them on demand depending what technology is used or even allow users writing their own and supplying them. 
I can refactor this once PR is merged.